meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -235,11 +235,13 @@ $divider-margin: 1.25rem;
 
 .filter-bar__search {
     display: inline;
-    width: calc(100% - 63px);
+    width: calc(100% - 37.1875px);
```

Comment:
the `magnifyier icon` seems to changed in width (for the projects search it looks better now as well)